validator/testcases/javascript/predefinedentities.py

```diff
@@ -922,6 +1020,14 @@ def interface_obj(iface):
             "perform unsafe operations. All ctypes use must be carefully "
             "reviewed by a qualified reviewer."),
         "editors_only": True,
+        "signing_help": ("Please try to avoid interacting with or bundling "
+                         "native binaries whenever possible. If you are "
+                         "bundling binaries for performance reasons, please "
+                         "consider alternatives such as Emscripten, "
+                         "JavaScript typed arrays, and Worker threads.",
+                         "Any code which makes use of the `ctypes` API "
+                         "must undergo at least one round of manual code "
+                         "review."),
```

Comment:
I thought about that. The last sentence is (importantly) different, so the first sentence could be reused. I mainly decided not to try because it's way to easy to trigger import loops in the validator at the moment.